packages/rxjs/integration/import/fixtures/browser/browser-test.js

```diff
@@ -0,0 +1,35 @@
+import { Observable, from, map } from './node_modules/rxjs/dist/esm/index.js';
```

Comment:
Just a rough set of imports and tests to assert that module imports are working okay.